__tests__/test.test.js

```diff
@@ -460,3 +460,85 @@ describe('GET /api/users', () => {
         })
     })
 })
+
+describe('GET /api/articles(queries)', () => {
+    test('Status code 200 returned', () => {
+        return request(app)
+        .get('/api/articles?sort_by=votes')
+        .expect(200)
+        })
+
+    test('Request responds with articles that have the correct properties and a comment_count', () => {
+        return request(app)
+        .get('/api/articles?sort_by=votes')
```

Comment:
What are we checking for with this test that's different to any previous tests? If it's to check that the query works we will need to test for the order the response is sorted in.